psydac/api/fem.py

```diff
@@ -715,17 +763,50 @@ def allocate_matrices(self, backend=None):
 
 #==============================================================================
 class DiscreteLinearForm(BasicDiscrete):
+    """ Class that represents the concept of a discrete linear form.
+        This class allocate the vector and generates the vector assembly method.
+
+    Parameters
+    ----------
+
+    expr : sympde.expr.expr.LinearForm
+        The symbolic bi-linear form.
+
+    kernel_expr : sympde.expr.evaluation.KernelExpression
+        The atomic representation of the linear form.
+
+    domain_h : Geometry
+        The discretized domain
+
+    space : FemSpace
+        The discrete spaces.
+
+    vector : Vector
+        The vector that we assemble into it.
+
+    update_ghost_regions : bool
+        Accumulate the contributions of the neighbouring processes.
+
+    quad_order : list or tuple
+        The number of quadrature points used in the assembly method.
+
+    backend : dict
+        The backend used to accelerate the computing kernels.
+
+    symbolic_mapping : Sympde.topology.Mapping
+        The symbolic mapping of the bi-linear form domain.
```

Comment:
Did you mean this?
```suggestion
        The symbolic mapping which defines the physical domain of the linear form.
```

Reply:
fixed in [9ed13cb](https://github.com/pyccel/psydac/pull/232/commits/9ed13cb7ef3cc0b2cf5ef6284c62b95ffd2874f1)